sdk/src/context.ts

```diff
@@ -21,6 +22,20 @@ export type WhirlpoolContextOpts = {
   userDefaultConfirmCommitment?: Commitment;
 };
 
+/**
+ * Default settings used when resolving token accounts.
+ * @category Core
+ */
+export type AccountResolverOptions = {
```

Comment:
can this be an optional (w/ default) paramter in WhirlpoolContextOpts?

Reply:
yeah, I've moved accountResolverOptions to WhirlpoolContextOpts.